k8s/demo/cassandra/README.md

```diff
@@ -0,0 +1,223 @@
+
+The number of replicas in the statefulset can be modified as required. This example makes use of 2 replicas.
+
+```
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: cassandra
+  labels:
+    app: cassandra
+spec:
+  serviceName: cassandra
+  replicas: 2
+  selector:
+    matchLabels:
+      app: cassandra
+  template:
+    metadata:
+      labels:
+        app: cassandra
+:
+```
+
+Execute the following commands:
+
+```
+test@Master:~$ cd openebs/k8s/demo/cassandra
+
+test@Master:~/openebs/k8s/demo/cassandra$ ls -ltr
+total 8
+-rw-rw-r-- 1 karthik karthik  165 Oct 30 12:19 cassandra-service.yaml
+-rw-rw-r-- 1 karthik karthik 2382 Nov 11 14:09 cassandra-statefulset.yaml
+```
+
+```
+test@Master:~/openebs/k8s/demo/cassandra$ kubectl apply -f cassandra-service.yaml
+service "cassandra" configured
+```
+
+```
+test@Master:~/openebs/k8s/demo/cassandra$ kubectl apply -f cassandra-statefulset.yaml
+statefulset "cassandra" created
+```
+
+Verify that all the OpenEBS persistent volumes are created, the Cassandra headless service and replicas 
+are up and running:
+
+```
+test@Master:~/openebs/k8s/demo/cassandra$ kubectl get pods
+NAME                                                             READY     STATUS    RESTARTS   AGE
+cassandra-0                                                      1/1       Running   0          4h
+cassandra-1                                                      1/1       Running   0          4h
+maya-apiserver-3416621614-g6tmq                                  1/1       Running   1          8d
+openebs-provisioner-4230626287-503dv                             1/1       Running   1          8d
+pvc-1c16536c-c6bc-11e7-a0eb-000c298ff5fc-ctrl-599202565-2kdff    1/1       Running   0          4h
+pvc-1c16536c-c6bc-11e7-a0eb-000c298ff5fc-rep-3068892500-22ccd    1/1       Running   0          4h
+pvc-1c16536c-c6bc-11e7-a0eb-000c298ff5fc-rep-3068892500-lhwdw    1/1       Running   0          4h
+pvc-e7d18817-c6bb-11e7-a0eb-000c298ff5fc-ctrl-1103031005-8vv82   1/1       Running   0          4h
+pvc-e7d18817-c6bb-11e7-a0eb-000c298ff5fc-rep-3006965094-cntx5    1/1       Running   0          4h
+pvc-e7d18817-c6bb-11e7-a0eb-000c298ff5fc-rep-3006965094-mhsjt    1/1       Running   0          4h
+```
+
+```
+test@Master:~/openebs/k8s/demo/cassandra$ kubectl get svc
+NAME                                                CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
+cassandra                                           None             <none>        9042/TCP            5h
+kubernetes                                          10.96.0.1        <none>        443/TCP             14d
+maya-apiserver-service                              10.102.92.217    <none>        5656/TCP            14d
+pvc-1c16536c-c6bc-11e7-a0eb-000c298ff5fc-ctrl-svc   10.107.177.156   <none>        3260/TCP,9501/TCP   4h
+pvc-d1fee1af-bf73-11e7-9be8-000c298ff5fc-ctrl-svc   10.98.87.196     <none>        3260/TCP,9501/TCP   9d
```

Comment:
should this pvc-d2fee1af and below df9e8.. be removed?

Reply:
Yes @kmova , these are residues from an older deployment involving a test pvc. Will remove this